deployment/gcp/README.md

```diff
@@ -104,7 +105,7 @@ Remember to change the `crawl.yaml` to point to `image: gcr.io/$PROJECT/openwpm`
 
 Launch a 1GB Basic tier Google Cloud Memorystore for Redis instance ($0.049/GB/hour):
 ```
-gcloud redis instances create crawlredis --size=1 --region=us-central1 --redis-version=redis_4_0
+gcloud redis instances create $CRAWL_PREFIX-redis --size=1 --region=us-central1 --redis-version=redis_4_0
 ```
 
 Launch a temporary redis-box pod deployed to the cluster which we use to interact with the above Redis instance:
```

Comment:
Should the redis box also be prefixed with the Crawlprefix?

Reply:
The redis box starts as a pod in the cluster, and the cluster already has a prefix. I don't think it's necessary.